.github/workflows/test-deploy-wf-service.yml

```
@@ -25,17 +25,185 @@
+        uses: actions/checkout@v4
+
+      - name: Get Latest Commit ID
+        id: lastcommit
+        uses: nmbgeek/github-action-get-latest-commit@main
+        with:
+          owner: ${{ github.repository_owner }}
+          repo: ballerine
+          branch: ${{ inputs.environment }}
+      
+      - name: Set Commit Id as Env 
+        run: echo "SHORT_HASH=${{ steps.lastcommit.outputs.shorthash }}" >> $GITHUB_ENV
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:64: Double quote to prevent globbing and word splitting

(shellcheck)